src/main/scala/com/cognite/sdk/scala/v1/resources/transformations.scala

```diff
@@ -113,6 +114,21 @@ class Transformations[F[_]](val requestSession: RequestSession[F])
       Constants.defaultBatchSize,
       aggregatedProperties
     )
+
+  def run(externalId: String): F[JobDetails] =
```

Comment:
we could also implement another run function with wait as a utility here, just an idea 😅

Reply:
like wait for the job to finish and return the status? 
Job can take time to run, not sure if we want to wait here?

Reply:
just as a separate function which polls and waits until job finishes as in python sdk but i think nvm. this is just for workflows so it is ok to scope what we support.